ind.ie.Gnomit.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>ind.ie.Gnomit.desktop</id>
+  <name>Gnomit</name>
```

Comment:
indenting!

Reply:
Thanks, Ned. Fixed in https://source.ind.ie/gnome/gnomit/gjs/commit/bcabb3d532c82348c2bc8b9522374d131d0bfdc4 – will include it in the next push to this PR once I handle some of the other issues also.